rep-0003.rst

```diff
@@ -281,6 +281,68 @@ Build System Support:
 
 - Same as Indigo
 
+Melodic Morenia (May 2018 - May 2023)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Artful (17.10)
+- Ubuntu B (18.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 28
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++11
+- Python 2.7
+
+  - Python >= 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.3.14
+
+Requirements:
+
+
++---------+---------------+----------------+----------------+-----------+
+| Package | Ubuntu Artful | Ubuntu B       | Debian Stretch | Fedora 28 |
++=========+===============+================+================+===========+
+|         |       Required Support         |    Recommended support     |
++---------+---------------+----------------+----------------+-----------+
+| Boost   |     1.62      |     1.62!      |    1.62        |   1.64!   |
++---------+---------------+----------------+----------------+-----------+
+| CMake   |     3.9.1     |     3.9.1!     |    3.7.2       |   3.9.3!  |
++---------+---------------+----------------+----------------+-----------+
+| Gazebo  |     7.5.0     |     7.5.0!     |    7.3.1       |   8.1.1!  |
++---------+---------------+----------------+----------------+-----------+
+| Ogre    |     1.9       |     1.9!       |    1.9         |   1.9!    |
++---------+---------------+----------------+----------------+-----------+
+| OpenCV  |     3.3*      |     3.3*       |    3.3*        |   3.3*    |
```

Comment:
Last year we forward ported the version of the last released ROS distro to avoid making assumption on what the community will release. I'd suggest putting 3.2 here and see if @vrabaud and/or other in the community releasing OpenCV in Fedora manifest interest in releasing 3.3.

Reply:
I was basing the 3.3 version mostly on this discussion: https://discourse.ros.org/t/opencv-3-3/2674/6 .  I guess we'll let @vrabaud comment and then we can change it/fix it accordingly.

Reply:
3.2 is from Dec 2016. It would be sad to be more than one year behind. 3.3 actually has very serious support for deep learning (not in contrib anymore) and I guess robotics will use more and more of that. 3.4 will be out in December 2017
I hate to say this as it will make me the maintainer for a few more years but we should go with our own 3.4 binaries ....

Reply:
@vrabaud Thanks for the response.  I'll change the PR to say 3.4 for now based on your recommendation.  If things change, we can change this later on.